content/en/docs/use-cases/openshift.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Using Flux on OpenShift
+linkTitle: OpenShift
+---
+
+## OpenShift Setup
+
+Steps described in this document has been tested on OpenShift 4.6 only. 
+You require cluster-admin privileged to install Flux on OpenShift. This means that it currently is not possiple to install Flux on the OpenShift Developer Sandbox.
```

Comment:
```suggestion
You require cluster-admin privileges to install Flux on OpenShift. This means that it currently is not possible to install Flux on the OpenShift Developer Sandbox.
```

Reply:
Does this need to be explained, e.g. what OpenShift Developer Sandbox is or why it's not possible, or is this clear to OS users?